src/scripts/main.js

```diff
@@ -1 +1,108 @@
 'use strict';
+
+let isClickedLeft = false;
+let isClickedRight = false;
+let timer;
+const notification = document.createElement('div');
+
+notification.setAttribute('data-qa', 'notification');
+
+function resetTimer() {
+  clearTimeout(timer);
+
+  timer = setTimeout(() => {
+    if (!isClickedLeft) {
+      firstPromise();
+    }
```

Comment:
The function `firstPromise()` is called without being defined. It seems like `firstPromise()` is a function that returns a promise. It is recommended to define this function before using it.